README.md

```diff
@@ -114,3 +114,14 @@ NAME       STATUS    AGE
 10.0.0.4   Ready     20s
 
 ```
+
+## Customization
+
+User have possibility to execute custom scripts on master before and after provision process.
+
+Scripts must be stored in directory: /usr/local/scripts/ with following names:
+
+- pre-tasks.sh - executed before provision
+- post-tasks.sh - executed after provision
+
+User must add those scripts to the image with correct rights: rw-r-r
```

Comment:
your opinion seems good.
This idea is generally used in Linux.

ex) /etc/profile
```
...
if [ -d /etc/profile.d ]; then
  for i in /etc/profile.d/*.sh; do
    if [ -r $i ]; then
      . $i
    fi
  done
  unset i
fi
```